scripts/magic/erase_box.sh

```diff
@@ -1,29 +1,31 @@
 #!/bin/bash
-
 : ${1?"Usage: $0 file.gds llx lly urx ury"}
 : ${2?"Usage: $0 file.gds llx lly urx ury"}
 : ${3?"Usage: $0 file.gds llx lly urx ury"}
 : ${4?"Usage: $0 file.gds llx lly urx ury"}
 : ${5?"Usage: $0 file.gds llx lly urx ury"}
 : ${PDK_ROOT?"You need to export PDK_ROOT"}
 
+cat <<HEREDOC > /dev/stderr
+====================
+WARNING
+====================
+erase_box.sh is deprecated and WILL be removed in a future version of OpenLane.
 
-export PDK=sky130A
+Please update your interactive scripts to use the built-in OpenLane function
+"erase_box" instead.
+====================
+HEREDOC
+
+MAGIC_SCRIPTS_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
+export PDK=sky130A
```

Comment:
is that needed here?

Reply:
That was the way it was in the original script, so yes.